Examples/CalloutExample.qml

```diff
@@ -94,7 +87,7 @@ Rectangle {
                     mapView.calloutData.location = clickedPoint;
                     mapView.calloutData.title = "Location";
                     mapView.calloutData.detail = "Lat: " + point.y.toFixed(2) + " Long: " + point.x.toFixed(2);
-                    mapView.calloutData.imageUrl = System.userHomePath + "/ArcGIS/Runtime/UnitTests/images/RedShinyPin.png";
+                    mapView.calloutData.imageUrl = System.userHomePath + "/images/RedShinyPin.png";
```

Comment:
@nandinirao This should just be `mapView.calloutData.imageUrl = "images/RedShinyPin.png";`
And then the RedShinyPin.png should be added to the images folder.